assisted-events-scrape/tests/integration/test_integration.py

```diff
@@ -0,0 +1,34 @@
+from config import ElasticsearchConfig
+import elasticsearch
+import waiting
+import time
+
+INIT_DELAY_SECONDS = 10
+ALL_EVENTS_NUMBER = 101
+
+
+class TestIntegration:
+    def setup(self):
+        self._wait_for_elastic()
```

Comment:
can we change to usage of ``pytest.fixture``?
that's the recommended way by pytest for setup/teardown actions (and for some other stuff)